oe_migration.info.yml

```diff
@@ -7,5 +7,3 @@ core: 8.x
 dependencies:
   - drupal:migrate
   - drupal:migrate_drupal
-  - migrate_plus:migrate_plus
```

Comment:
This dependency will be declared in a sub-module.

Reply:
True. But you can declare all the contrib dependencies needed for submodules in the composer.json --dev so that they get installed when doing development. Then you can add them to the README.md of the submodule to inform the users that if they wanna use it, they need to require that dependency

Reply:
Thank you very much for this very useful comment, I'll surely keep this in mind.

Maybe it should be better to declare, for the moment at least, the absolutely necessary dependencies and update `composer.json` and the `info` file, if needed, as we submit more code. This way the newly declared dependency will be part of the related PR.

I'll remove, for the same reason, the `drupal:migrate_drupal` dependency.